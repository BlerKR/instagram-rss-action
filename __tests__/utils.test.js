const { formatFeed, generateFeed } = require("../src/utils");
const feed = require("./fixtures/feed.json");
const hashtags = require("./fixtures/hashtags.json");
const video = require("./fixtures/video.json");

const metadata = {
  title: "Insta brunch",
};

it("formatFeed", () => {
  expect(formatFeed(feed, "katydecorah")).toMatchSnapshot();
  expect(formatFeed(hashtags, "eddiefs_eatery")).toMatchSnapshot();
  expect(formatFeed(video, "tatutacony")).toMatchSnapshot();
});

it("generateFeed", () => {
  const posts = formatFeed(feed, "katydecorah");
  expect(generateFeed(posts, metadata)).toMatchSnapshot();
});
