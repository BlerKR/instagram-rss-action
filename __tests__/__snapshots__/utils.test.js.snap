// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formatFeed 1`] = `
Array [
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Today I paid off my student loan debt.

Cake by @twobirdsmarketplace</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/123245645_1059345254525114_1611454811250817146_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=oxSmPyA9_5YAX-CxjsG&tp=1&oh=3606af6867b561b2726ea48d04ab197b&oe=605AA0A6\\" alt=\\"\\" /></p>",
    "date_published": "2020-10-30T13:54:05.000Z",
    "id": "2431404674086406779",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/123245645_1059345254525114_1611454811250817146_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=oxSmPyA9_5YAX-CxjsG&tp=1&oh=3606af6867b561b2726ea48d04ab197b&oe=605AA0A6",
    "title": "Today I paid off my student loan debt",
    "url": "https://instagram.com/p/CG-FatAJQp7",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Apple sheet cake</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71886735_532705957528329_6522322102253595207_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=E0MkbpfuYI4AX8NtgkN&tp=1&oh=fb9cb333923053d1d315732dc19913de&oe=605B15AC\\" alt=\\"\\" /></p>",
    "date_published": "2019-11-09T20:38:32.000Z",
    "id": "2173588080672540774",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71886735_532705957528329_6522322102253595207_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=E0MkbpfuYI4AX8NtgkN&tp=1&oh=fb9cb333923053d1d315732dc19913de&oe=605B15AC",
    "title": "Apple sheet cake",
    "url": "https://instagram.com/p/B4qItfcgohm",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>We are monster foodies!</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72291222_178059880030486_627605298652157995_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=110&_nc_ohc=id6q5JP3jIIAX86a8PZ&tp=1&oh=391605f65cb77bbf95b48c3931631e8f&oe=605B72DE\\" alt=\\"\\" /></p>",
    "date_published": "2019-10-31T14:50:34.000Z",
    "id": "2166889962656668071",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72291222_178059880030486_627605298652157995_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=110&_nc_ohc=id6q5JP3jIIAX86a8PZ&tp=1&oh=391605f65cb77bbf95b48c3931631e8f&oe=605B72DE",
    "title": "We are monster foodies",
    "url": "https://instagram.com/p/B4SVvC7gpmn",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>☾ bitchin</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72786297_2365693070426381_5929100958132233099_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=105&_nc_ohc=O1ClU-QyTlgAX8SON4c&tp=1&oh=1b1dae76e57c7b9f80fa9a14c0a59bb0&oe=6059B729\\" alt=\\"\\" /></p>",
    "date_published": "2019-10-25T20:47:14.000Z",
    "id": "2162720822165245808",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72786297_2365693070426381_5929100958132233099_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=105&_nc_ohc=O1ClU-QyTlgAX8SON4c&tp=1&oh=1b1dae76e57c7b9f80fa9a14c0a59bb0&oe=6059B729",
    "title": "☾ bitchin",
    "url": "https://instagram.com/p/B4DhyERgA9w",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Choux donuts</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71190752_441886183108477_7988142840093531655_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=jNRgxmLctxEAX9PnsXr&tp=1&oh=8ff545c25ba16a074670595b55f6f0ea&oe=605A3C4F\\" alt=\\"\\" /></p>",
    "date_published": "2019-10-14T19:52:07.000Z",
    "id": "2154720547676741908",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71190752_441886183108477_7988142840093531655_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=jNRgxmLctxEAX9PnsXr&tp=1&oh=8ff545c25ba16a074670595b55f6f0ea&oe=605A3C4F",
    "title": "Choux donuts",
    "url": "https://instagram.com/p/B3nGuwCgeEU",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Rose apple tart. Recipe by @yossyarefi.</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72895211_778207925931601_2112736808330979178_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=-1kX6JB8XbIAX92I03L&tp=1&oh=4106ce9932c902d0801654036d164f36&oe=6059C5F3\\" alt=\\"\\" /></p>",
    "date_published": "2019-10-07T12:43:22.000Z",
    "id": "2149431320818773784",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72895211_778207925931601_2112736808330979178_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=-1kX6JB8XbIAX92I03L&tp=1&oh=4106ce9932c902d0801654036d164f36&oe=6059C5F3",
    "title": "Rose apple tart",
    "url": "https://instagram.com/p/B3UUGXqgG8Y",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/70737581_105544737401545_1399334489263432147_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=ftxnNQMEcXQAX-9U8Jo&tp=1&oh=71b53612610609aa42235ce4423dccc9&oe=605B8939\\" alt=\\"\\" /></p>",
    "date_published": "2019-09-16T01:11:50.000Z",
    "id": "2133862968882684736",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/70737581_105544737401545_1399334489263432147_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=ftxnNQMEcXQAX-9U8Jo&tp=1&oh=71b53612610609aa42235ce4423dccc9&oe=605B8939",
    "title": "",
    "url": "https://instagram.com/p/B2dARCDnONA",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67841550_724311197981702_6136181798484813957_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=SpmJhwsZ2CgAX__DT9j&tp=1&oh=1e7264b5781200c34fe7d170193a6db7&oe=605A3365\\" alt=\\"\\" /></p>",
    "date_published": "2019-09-08T11:42:36.000Z",
    "id": "2128382238700733651",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67841550_724311197981702_6136181798484813957_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=SpmJhwsZ2CgAX__DT9j&tp=1&oh=1e7264b5781200c34fe7d170193a6db7&oe=605A3365",
    "title": "",
    "url": "https://instagram.com/p/B2JiF6UAgTT",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/67681756_2715739055158177_1478089007444277229_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=wg-A_Av2EzUAX_Yinty&tp=1&oh=8aafcb2881168bfde983bd0a0f23c356&oe=605A0B66\\" alt=\\"\\" /></p>",
    "date_published": "2019-09-07T20:00:15.000Z",
    "id": "2127907940858670468",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/67681756_2715739055158177_1478089007444277229_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=wg-A_Av2EzUAX_Yinty&tp=1&oh=8aafcb2881168bfde983bd0a0f23c356&oe=605A0B66",
    "title": "",
    "url": "https://instagram.com/p/B2H2P-BnS2E",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Jason and I are in a Times Union article today about working from home. (Photos by Will Waldron)</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/69425380_146041686601701_4846739388620734892_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=uh9R92ZV-8MAX_yxPHc&tp=1&oh=00ee9e8e1ba6985438a5ace068341440&oe=605AA5DF\\" alt=\\"\\" /></p>",
    "date_published": "2019-08-31T16:48:59.000Z",
    "id": "2122738241866893757",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/69425380_146041686601701_4846739388620734892_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=uh9R92ZV-8MAX_yxPHc&tp=1&oh=00ee9e8e1ba6985438a5ace068341440&oe=605AA5DF",
    "title": "Jason and I are in a Times Union article today about working from home",
    "url": "https://instagram.com/p/B11ey8pnQm9",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>Cowboy cookie ice cream. (My first try at writing a dairy-free recipe and my first time making granola. Due to my reflection in the scoop, this is also a selfie.)</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67178229_1274041706103256_5381338537786988100_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=104&_nc_ohc=Pz0NePOA5boAX_rjFuM&tp=1&oh=e2fd356bef386767eaa95304b3587dd4&oe=605AAE8D\\" alt=\\"\\" /></p>",
    "date_published": "2019-08-24T18:07:04.000Z",
    "id": "2117704113792970888",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67178229_1274041706103256_5381338537786988100_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=104&_nc_ohc=Pz0NePOA5boAX_rjFuM&tp=1&oh=e2fd356bef386767eaa95304b3587dd4&oe=605AAE8D",
    "title": "Cowboy cookie ice cream",
    "url": "https://instagram.com/p/B1jmKviA3CI",
  },
  Object {
    "authors": Array [
      Object {
        "name": "katydecorah",
      },
    ],
    "content_html": "<p>🌱</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67514469_105482600715402_358065942120119909_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=106&_nc_ohc=167sU8iYWb0AX_Jr1r5&tp=1&oh=39f9986052fdd8841f2a82fcf303af5f&oe=605BEA07\\" alt=\\"\\" /></p>",
    "date_published": "2019-08-23T22:52:03.000Z",
    "id": "2117122775289202996",
    "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67514469_105482600715402_358065942120119909_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=106&_nc_ohc=167sU8iYWb0AX_Jr1r5&tp=1&oh=39f9986052fdd8841f2a82fcf303af5f&oe=605BEA07",
    "title": "🌱",
    "url": "https://instagram.com/p/B1hh_J3AIk0",
  },
]
`;

exports[`generateFeed 1`] = `
Object {
  "favicon": "https://katydecorah.com/favicon.ico",
  "feed_url": "https://katydecorah.com/instagram-rss/feed.json",
  "home_page_url": "https://katydecorah.com/",
  "items": Array [
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Today I paid off my student loan debt.

Cake by @twobirdsmarketplace</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/123245645_1059345254525114_1611454811250817146_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=oxSmPyA9_5YAX-CxjsG&tp=1&oh=3606af6867b561b2726ea48d04ab197b&oe=605AA0A6\\" alt=\\"\\" /></p>",
      "date_published": "2020-10-30T13:54:05.000Z",
      "id": "2431404674086406779",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/123245645_1059345254525114_1611454811250817146_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=oxSmPyA9_5YAX-CxjsG&tp=1&oh=3606af6867b561b2726ea48d04ab197b&oe=605AA0A6",
      "title": "Today I paid off my student loan debt",
      "url": "https://instagram.com/p/CG-FatAJQp7",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Apple sheet cake</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71886735_532705957528329_6522322102253595207_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=E0MkbpfuYI4AX8NtgkN&tp=1&oh=fb9cb333923053d1d315732dc19913de&oe=605B15AC\\" alt=\\"\\" /></p>",
      "date_published": "2019-11-09T20:38:32.000Z",
      "id": "2173588080672540774",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71886735_532705957528329_6522322102253595207_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=E0MkbpfuYI4AX8NtgkN&tp=1&oh=fb9cb333923053d1d315732dc19913de&oe=605B15AC",
      "title": "Apple sheet cake",
      "url": "https://instagram.com/p/B4qItfcgohm",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>We are monster foodies!</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72291222_178059880030486_627605298652157995_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=110&_nc_ohc=id6q5JP3jIIAX86a8PZ&tp=1&oh=391605f65cb77bbf95b48c3931631e8f&oe=605B72DE\\" alt=\\"\\" /></p>",
      "date_published": "2019-10-31T14:50:34.000Z",
      "id": "2166889962656668071",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72291222_178059880030486_627605298652157995_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=110&_nc_ohc=id6q5JP3jIIAX86a8PZ&tp=1&oh=391605f65cb77bbf95b48c3931631e8f&oe=605B72DE",
      "title": "We are monster foodies",
      "url": "https://instagram.com/p/B4SVvC7gpmn",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>☾ bitchin</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72786297_2365693070426381_5929100958132233099_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=105&_nc_ohc=O1ClU-QyTlgAX8SON4c&tp=1&oh=1b1dae76e57c7b9f80fa9a14c0a59bb0&oe=6059B729\\" alt=\\"\\" /></p>",
      "date_published": "2019-10-25T20:47:14.000Z",
      "id": "2162720822165245808",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72786297_2365693070426381_5929100958132233099_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=105&_nc_ohc=O1ClU-QyTlgAX8SON4c&tp=1&oh=1b1dae76e57c7b9f80fa9a14c0a59bb0&oe=6059B729",
      "title": "☾ bitchin",
      "url": "https://instagram.com/p/B4DhyERgA9w",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Choux donuts</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71190752_441886183108477_7988142840093531655_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=jNRgxmLctxEAX9PnsXr&tp=1&oh=8ff545c25ba16a074670595b55f6f0ea&oe=605A3C4F\\" alt=\\"\\" /></p>",
      "date_published": "2019-10-14T19:52:07.000Z",
      "id": "2154720547676741908",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/71190752_441886183108477_7988142840093531655_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=jNRgxmLctxEAX9PnsXr&tp=1&oh=8ff545c25ba16a074670595b55f6f0ea&oe=605A3C4F",
      "title": "Choux donuts",
      "url": "https://instagram.com/p/B3nGuwCgeEU",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Rose apple tart. Recipe by @yossyarefi.</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72895211_778207925931601_2112736808330979178_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=-1kX6JB8XbIAX92I03L&tp=1&oh=4106ce9932c902d0801654036d164f36&oe=6059C5F3\\" alt=\\"\\" /></p>",
      "date_published": "2019-10-07T12:43:22.000Z",
      "id": "2149431320818773784",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/72895211_778207925931601_2112736808330979178_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=-1kX6JB8XbIAX92I03L&tp=1&oh=4106ce9932c902d0801654036d164f36&oe=6059C5F3",
      "title": "Rose apple tart",
      "url": "https://instagram.com/p/B3UUGXqgG8Y",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/70737581_105544737401545_1399334489263432147_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=ftxnNQMEcXQAX-9U8Jo&tp=1&oh=71b53612610609aa42235ce4423dccc9&oe=605B8939\\" alt=\\"\\" /></p>",
      "date_published": "2019-09-16T01:11:50.000Z",
      "id": "2133862968882684736",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/70737581_105544737401545_1399334489263432147_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=ftxnNQMEcXQAX-9U8Jo&tp=1&oh=71b53612610609aa42235ce4423dccc9&oe=605B8939",
      "title": "",
      "url": "https://instagram.com/p/B2dARCDnONA",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67841550_724311197981702_6136181798484813957_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=SpmJhwsZ2CgAX__DT9j&tp=1&oh=1e7264b5781200c34fe7d170193a6db7&oe=605A3365\\" alt=\\"\\" /></p>",
      "date_published": "2019-09-08T11:42:36.000Z",
      "id": "2128382238700733651",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67841550_724311197981702_6136181798484813957_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=102&_nc_ohc=SpmJhwsZ2CgAX__DT9j&tp=1&oh=1e7264b5781200c34fe7d170193a6db7&oe=605A3365",
      "title": "",
      "url": "https://instagram.com/p/B2JiF6UAgTT",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/67681756_2715739055158177_1478089007444277229_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=wg-A_Av2EzUAX_Yinty&tp=1&oh=8aafcb2881168bfde983bd0a0f23c356&oe=605A0B66\\" alt=\\"\\" /></p>",
      "date_published": "2019-09-07T20:00:15.000Z",
      "id": "2127907940858670468",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/67681756_2715739055158177_1478089007444277229_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=101&_nc_ohc=wg-A_Av2EzUAX_Yinty&tp=1&oh=8aafcb2881168bfde983bd0a0f23c356&oe=605A0B66",
      "title": "",
      "url": "https://instagram.com/p/B2H2P-BnS2E",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Jason and I are in a Times Union article today about working from home. (Photos by Will Waldron)</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/69425380_146041686601701_4846739388620734892_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=uh9R92ZV-8MAX_yxPHc&tp=1&oh=00ee9e8e1ba6985438a5ace068341440&oe=605AA5DF\\" alt=\\"\\" /></p>",
      "date_published": "2019-08-31T16:48:59.000Z",
      "id": "2122738241866893757",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/69425380_146041686601701_4846739388620734892_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=109&_nc_ohc=uh9R92ZV-8MAX_yxPHc&tp=1&oh=00ee9e8e1ba6985438a5ace068341440&oe=605AA5DF",
      "title": "Jason and I are in a Times Union article today about working from home",
      "url": "https://instagram.com/p/B11ey8pnQm9",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>Cowboy cookie ice cream. (My first try at writing a dairy-free recipe and my first time making granola. Due to my reflection in the scoop, this is also a selfie.)</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67178229_1274041706103256_5381338537786988100_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=104&_nc_ohc=Pz0NePOA5boAX_rjFuM&tp=1&oh=e2fd356bef386767eaa95304b3587dd4&oe=605AAE8D\\" alt=\\"\\" /></p>",
      "date_published": "2019-08-24T18:07:04.000Z",
      "id": "2117704113792970888",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67178229_1274041706103256_5381338537786988100_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=104&_nc_ohc=Pz0NePOA5boAX_rjFuM&tp=1&oh=e2fd356bef386767eaa95304b3587dd4&oe=605AAE8D",
      "title": "Cowboy cookie ice cream",
      "url": "https://instagram.com/p/B1jmKviA3CI",
    },
    Object {
      "authors": Array [
        Object {
          "name": "katydecorah",
        },
      ],
      "content_html": "<p>🌱</p><p><img src=\\"https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67514469_105482600715402_358065942120119909_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=106&_nc_ohc=167sU8iYWb0AX_Jr1r5&tp=1&oh=39f9986052fdd8841f2a82fcf303af5f&oe=605BEA07\\" alt=\\"\\" /></p>",
      "date_published": "2019-08-23T22:52:03.000Z",
      "id": "2117122775289202996",
      "image": "https://scontent-lga3-2.cdninstagram.com/v/t51.2885-15/e35/s1080x1080/67514469_105482600715402_358065942120119909_n.jpg?_nc_ht=scontent-lga3-2.cdninstagram.com&_nc_cat=106&_nc_ohc=167sU8iYWb0AX_Jr1r5&tp=1&oh=39f9986052fdd8841f2a82fcf303af5f&oe=605BEA07",
      "title": "🌱",
      "url": "https://instagram.com/p/B1hh_J3AIk0",
    },
  ],
  "title": "Instagram feed",
  "version": "https://jsonfeed.org/version/1.1",
}
`;
